2020-02-Spring/2_linked_lists/README.md

```diff
@@ -45,22 +45,43 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. Remove Zero Sum Consecutive Nodes from Linked List
 
-Source: TODO :bug:
+Source: https://leetcode.com/problems/remove-zero-sum-consecutive-nodes-from-linked-list/
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given the head of a linked list, we repeatedly delete consecutive sequences of nodes that sum to 0 until there are no such sequences.
 
-#### Example Input
+After doing so, return the head of the final linked list.  You may return any such answer.
 
-If the problem is simple enough, remove this section. TODO :bug:
+(Note that in the examples below, all sequences are serializations of ListNode objects.)
 
+#### Example Input
+##### Example 1: <br>
```

Comment:
the HTML `<br>` is fine, but shouldn't be necessary. When in doubt, include whitespace between paragraphs and headings instead of using HTML.